src/index.js

```diff
@@ -1,5 +1,23 @@
 import './style.css';
-import data from './data.js';
-import Input from './Input.js';
+import input from './Input.js';
+import { getScores, addScore } from './api.js';
 
-Input(data);
\ No newline at end of file
+const btn = document.querySelector('.submit-btn');
+
+async function mainRender() {
```

Comment:
- [ ] Use ES6 syntax for your function.

Reply:
Thank you,
Will change the function to arrow function as requested.